<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrarion</string>
  </property>
  <widget class="QLabel" name="registrationLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="repeatPasswordLine"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="loginLine"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="loginLabel">
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordLine"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="repeatPasswordLabel">
      <property name="text">
       <string>Repeat Password:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Passwords are not equal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signInButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>Return to Authorization Window</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLine</tabstop>
  <tabstop>loginLine</tabstop>
  <tabstop>passwordLine</tabstop>
  <tabstop>repeatPasswordLine</tabstop>
  <tabstop>signUpButton</tabstop>
  <tabstop>signInButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
